### PR TITLE
Add default page command

### PR DESCRIPTION
**Pull request by akmiller01**

### CODE REVIEW

**Comment by akmiller01** [comment]
Complete, but could use some comments on the command, maybe...

File: iati/home/templatetags/iati_tags.py [hunk 1]
```diff
@@ -1,15 +1,35 @@
...
 
 register = template.Library()
 
 @register.simple_tag(takes_context=True)
-def events_index_page_url(context):
-    """Returns the relative url for the events index page"""
-    events_index_page = EventIndexPage.objects.live().first()
-    if events_index_page is not None:
-        return events_index_page.get_url(context['request'])
+def default_page_url(context, default_page_name="home"):
```
Comment (allthatilk) [inline iati/home/templatetags/iati_tags.py]: This code is really WET. Please DRY it up.

File: iati/home/management/commands/create_default_pages.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+from contact.models import ContactPage
+from events.models import EventIndexPage
+from guidance_and_support.models import GuidanceAndSupportPage
+from news.models import NewsIndexPage
+
+from django.conf import settings
+
+class Command(BaseCommand):
+    help = 'Create the default pages that constitute the skeleton of the website information architecture'
+    def handle(self, *args, **options):
```
Comment (allthatilk) [inline iati/home/management/commands/create_default_pages.py]: This needs a docstring

File: iati/home/management/commands/create_default_pages.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+            home_page.title_en = "Home"
+            home_page.slug_en = "home"
+            home_page.url_path_en = "/home/"
+            home_page.title = "Home"
+            home_page.slug = "home"
+            home_page.url_path = "/home/"
+
+            self.stdout.write(self.style.SUCCESS('Successfully fixed home page...'))
+
+            about_page = AboutPage.objects.live().first()
```
Comment (allthatilk) [inline iati/home/management/commands/create_default_pages.py]: not sure if this is intentionally nested or not?

**Pull request merged**